.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish
+
+on:
+  release:
+    types: [ created ]
```

Comment:
esse created vale pra branch tb ? Acho que fica perigoso se for isso
```
A Git branch or tag is created. For more information, see the "Git data" REST API.
```

Reply:
Isso só vale pra `release`